backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/inventorsoftacademy
```

Comment:
Please, move environment specific properties into separate profile config file (url, username and password will changed between environments, like your local machine and test server. therefore we don't want to put them in generic config)